fendermint/app/src/cmd/genesis.rs

```diff
@@ -187,6 +194,23 @@ where
     Ok(())
 }
 
+fn set_eam_permissions(
+    genesis_file: &PathBuf,
+    args: &GenesisSetEAMPermissionsArgs,
+) -> anyhow::Result<()> {
+    update_genesis(genesis_file, |mut genesis| {
+        genesis.eam_permission_mode = match args.mode.to_lowercase().as_str() {
+            "unrestricted" => PermissionMode::Unrestricted,
```

Comment:
Not a blocker for this PR but should we use `strum` in the same way as in https://github.com/consensus-shipyard/ipc/pull/691 ?

Reply:
I don't think it's worth it, as these are more complex enums.